vagrant_env/Vagrantfile

```diff
@@ -0,0 +1,56 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Requires vagrant-host-shell and vagrant-vbguest
+required_plugins = %w(
+    vagrant-host-shell
+    vagrant-sshfs
+    vagrant-vbguest
+    )
+required_plugins.each do |plugin|
+  system "vagrant plugin install #{plugin}" unless Vagrant.has_plugin? plugin
+end
+
+MEM = 2048
+CPU = 2
+
+VAGRANTFILE_API_VERSION = '2'
+#
+#  Fully documented Vagrantfile available
+#  in the wiki:  https://github.com/josenk/vagrant-vmware-esxi/wiki
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+    config.vm.synced_folder "./", "/vagrant"
+    config.vm.synced_folder "../", "/home/vagrant/rockstor-installer"
+    config.vm.network "private_network", type: "dhcp"
+    config.env.enable # Enable vagrant-env(.env)
+
+    config.vm.define "rockstor-installer" do |v|
+        v.vm.hostname = "rockstor-installer"
+        v.vm.box = 'bento/opensuse-leap-15'
+
+        # Provide specific variable
+        v.vm.provider :virtualbox do |vb|
+            vb.memory = MEM
+            vb.cpus = CPU
+        end
+
+        config.vm.provision "shell", inline: <<-SHELL
+            PROFILE=x86_64
+            #PROFILE=AArch64
+            REPO_URL="https://github.com/rockstor/rockstor-installer.git"
+            REPO_DIR="rockstor-installer/"
+            if [ "$PROFILE" = "x86_64" ]; then
+                sudo zypper addrepo http://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder/openSUSE_Leap_15.2/ appliance-builder
+            else
+                sudo zypper addrepo http://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder/openSUSE_Leap_15.2_ARM/ appliance-builder
+            fi
+            sudo zypper -n install git
```

Comment:
```suggestion
            if [ "$PROFILE" = "x86_64" ]; then
                sudo zypper --non-interactive addrepo --refresh http://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder/openSUSE_Leap_15.2/ appliance-builder
            else
                sudo zypper --non-interactive addrepo --refresh http://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder/openSUSE_Leap_15.2_ARM/ appliance-builder
            fi
            sudo zypper --non-interactive --gpg-auto-import-keys refresh
            sudo zypper -n install git
```
This is the change I managed to delete last night while submitting my first review (sorry about that)!!!
From the following:
```
rockstor-installer: Do you want to reject the key, trust temporarily, or trust always? [r/t/a/?] (r): r                                                               
   rockstor-installer: Warning: Skipping repository 'appliance-builder' because of the above error.                                                                      
   rockstor-installer: Loading repository data...                                                                                                                        
   rockstor-installer: Error building the cache:
   rockstor-installer: [appliance-builder|http://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder/openSUSE_Leap_15.2/] Valid metadata not found at specified URL                                                                                                                                                            
   rockstor-installer: History:                                                                                                                                          
   rockstor-installer:  - Signature verification failed for repomd.xml                                                                                                   
   rockstor-installer:  - Can't provide /repodata/repomd.xml                                                                                                             
   rockstor-installer:                                                                                                                                                   
   rockstor-installer: Some of the repositories have not been refreshed because of an error.                                                                             
   rockstor-installer: Reading installed packages...
   rockstor-installer: Resolving package dependencies...
   rockstor-installer: The following 101 NEW packages are going to be installed:
```
It looks like the version of kiwi-ng installed, and consequently used, is actually the distro default one, not the newer one intended from the added repo.

Sorry but I haven't tested these changes just yet as called away currently. Do please report here if you find this fixes the suspected issue of the new repo not being used.

Reply:
Now I feel a bit daft having put the -n on other zipper calls and missed it on those. [Shame face]

Added a couple of minor corrections in the light of this and added to the README too.